app/scripts/views/support/create_secure_password.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const View = require('views/base').extend({
+    className: 'create_secure_password',
+    template: require('stache!templates/support/create_secure_password'),
+    afterRender: function () {
+      this.window.document.cookie = 'canGoBack=1; path=' + this.window.location.pathname;
```

Comment:
Do you think it would be worthwhile to add a test for this view? Similar to [clear storage](https://github.com/mozilla/fxa-content-server/blob/master/app/tests/spec/views/clear_storage.js#L30), but check cookie?